### PR TITLE
UIREQ-613: update request details pane

### PR DESCRIPTION
**Pull request by pavelspichonak**
## Purpose
Update request details pane

## Refs
https://issues.folio.org/browse/UIREQ-613

### CODE REVIEW

File: src/routes/RequestsRoute.js [hunk 1]
```diff
@@ -540,10 +545,14 @@ class RequestsRoute extends React.Component {
 
   // Called as a map function
   addRequestFields(r) {
+    const { requestLevel } = r;
+
     return Promise.all(
       [
         this.findResource('user', r.requesterId),
-        this.findResource('requestsForItem', r.itemId),
+        requestLevel === REQUEST_LEVEL_TYPES.ITEM
```
Comment (Stanislau-Snapok) [inline src/routes/RequestsRoute.js:553]: Could you please double-check should we show requests for `Instance`, if we working with `Item` level request

File: src/ViewRequest.js [hunk 2]
```diff
@@ -43,11 +43,12 @@ import { IfPermission } from '@folio/stripes/core';
 
 import CancelRequestDialog from './CancelRequestDialog';
 import ItemDetail from './ItemDetail';
+import TitleInformation from './components/TitleInformation';
 import UserDetail from './UserDetail';
 import RequestForm from './RequestForm';
 import PositionLink from './PositionLink';
 import MoveRequestManager from './MoveRequestManager';
-import { requestStatuses } from './constants';
+import { requestStatuses, REQUEST_LEVEL_TYPES } from './constants';
```
[the remark this answers is not in the document]
Comment (pavelspichonak) [reply src/ViewRequest.js]: done

File: src/ViewRequest.js [hunk 3]
```diff
@@ -528,23 +530,50 @@ class ViewRequest extends React.Component {
             id="item-info"
```
[the remark this answers is not in the document]
Comment (pavelspichonak) [reply src/ViewRequest.js]: Fixed. Thanks!

File: src/routes/RequestsRoute.js (hunk 1 above)
Comment (pavelspichonak) [reply src/routes/RequestsRoute.js]: done

**Review by Stanislau-Snapok — APPROVED** [review]

File: src/PositionLink.js [hunk 4]
```diff
@@ -5,11 +5,13 @@ import { Link } from 'react-router-dom';
...
 
 import { openRequestStatusFilters } from './utils';
+import { REQUEST_LEVEL_TYPES } from './constants';
 
 export default function PositionLink({ request }) {
   const queuePosition = get(request, 'position');
   const barcode = get(request, 'item.barcode');
-  const openRequestsPath = `/requests?filters=${openRequestStatusFilters}&query=${request.itemId}&sort=Request Date`;
+  const id = request[request.requestLevel === REQUEST_LEVEL_TYPES.ITEM ? 'itemId' : 'instanceId'];
+  const openRequestsPath = `/requests?filters=${openRequestStatusFilters}&query=${id}&sort=Request Date`;
```
Comment (Dmitriy-Litvinenko) [inline src/PositionLink.js:14]: Could you please move to const `const REQUEST_DATE = 'Request Date';` ?
the same for `export const getURL = (id, count) => <Link to={`/requests?filters=query=${id}&sort=Request Date`}>{count}</Link>;`

**Review by Dmitriy-Litvinenko — APPROVED** [review]

File: src/PositionLink.js (hunk 4 above)
Comment (pavelspichonak) [reply src/PositionLink.js]: Done

**Review by mkuklis — APPROVED** [review]

**Pull request merged**